### PR TITLE
Michelle committees

### PR DESCRIPTION
**Pull request by morden35**
Added scrape_committees(), get_committee_members(), get_thomas_mapping(), and fetch_current_committees() to convert_us.py. get_committee_members() makes a request to the committee membership url. get_thomas_mapping() maps committee names/chambers to thomas_ids. fetch_current_committees() makes a request to the current committees url and yields Committee() objects. @jamesturk

### CODE REVIEW

File: openstates/models/committees.py [hunk 1]
```diff
@@ -45,7 +45,7 @@ def validate_parent_and_classification(
     ) -> dict[str, typing.Any]:
         if (
             values.get("classification") == "subcommittee"
-            and values.get("parent") in COMMITTEE_PARENTS
+            and values.get("parent") not in COMMITTEE_PARENTS
```
Comment (jamesturk) [inline openstates/models/committees.py:48]: in retrospect this code was correct, I misunderstood where it was

committees should have a COMMITTEE_PARENT but subcommittee parents should be the name of the parent :)

**Review by jamesturk — CHANGES_REQUESTED** [review]
looking good

also when you're ready/done testing, please remove stray comments of non-used code and the TODOs (OK to replace them with actual comments, but let's at least get rid of the TODO part)

**Comment by jamesturk** [comment]
looks good, gave it a run & it looks like there are a bunch of (sub?)committees without members
 e.g. Western-Hemisphere-Civilian-Security-and-Trade 

could you verify that those do not have members in the source data (and on congress.gov?) and if not let's add a check before calling dump_obj that the committee has at least one member?  I think that'll keep this a bit more manageable for now and I don't see much value in us collecting committees without members in this case

**Comment by jamesturk** [comment]
also there are a few linter errors left:

`poetry run flake8 openstates/cli` will give feedback on those

if you want to learn a little about type checking
`poetry run mypy openstates/cli` will do the same for those (glad to tackle these together if it helps!)

**Comment by morden35** [comment]
> looks good, gave it a run & it looks like there are a bunch of (sub?)committees without members
> e.g. Western-Hemisphere-Civilian-Security-and-Trade
> 
> could you verify that those do not have members in the source data (and on congress.gov?) and if not let's add a check before calling dump_obj that the committee has at least one member? I think that'll keep this a bit more manageable for now and I don't see much value in us collecting committees without members in this case

**Pull request merged**